README.md
# Spirit

Share the secrets to healthy living and a better quality of life with Spirit – WordPress Wellness Theme, a high-end profile theme that balances inspiring images, clean typography and custom Upfront design. Connect new users to your yoga class, clean eating recipes or life coaching service.
 
> Whether you’re providing personal training or promoting a new organic cookbook, Spirit WordPress Wellness Theme offers a professional design with an honest, personal touch.

## A Clean One-Page Profile

Set your logo and share your skillset over the top of a stunning responsive header image. Highlight your services, special events and group sessions.

Add as many elements as you like, and promote your services with customizable calls to action.

## Pre-Formatted and Simple Customization

Showcase your work on a single, scrollable page, complete with stylish sticky navigation. Use any of the included and uniquely-styled template pages to highlight your location, products and services.

## A Clear, Readable Font

Open Sans was selected for its humanist design and open, upright letter form.

Its neutral, yet friendly appearance is optimized for web and mobile interfaces, and provides excellent legibility.

## Get Social

Spirit includes a refreshing and simple blog layout. Plus, you can link users to your Facebook fan page or inspire your clients with tips and advice on Twitter.

Give your members more ways to connect through custom social share icons. Keep your followers on track and help them meet their wellness goals with a well-rounded social plan.

## It’s All About You

The unique column layout of Spirit’s About page makes it easy to post your personal bio as well as quotes, testimonials, certificates and other qualifications.

You can make it even more personal with a well-placed profile picture, too.

## Contact Page

￼
Give people a simple way to find out more.
Contact Page
A simple email form over a beautiful background image makes it easy for users to provide feedback, ask questions or share a testimony of their success.

With Upfront it’s easy to add your physical location, phone number and more.

## eCommerce

Spirit comes styled MarketPress and WooCommerce ready so you can get an online store up and running in minutes.

Sell shirts, downloadable nutrition guides, yoga mats or whatever your clients need to be a success.

## Testimonials

Client success stories and reviews offer a powerful way to build your reputation. The pre-styled Testimonial page makes it easy to share how others have benefited from your services.

Capture leads and invite others to join the wellness revolution with you by joining your mailing list.

## Coastal Color Palate

Colors inspired by ocean waves and sandy beaches compliment just about any image.

Spirit provides space for wall-to-wall background images, huge headers and a custom built sortable gallery with subtle animations. Plus, there’s light box for a distraction-free way to view your images.

## CUSTOMIZATION

Take complete control over colors, fonts, styles, images, videos and content - in just a few clicks.

## RESPONSIVE

Upfront makes it easy to edit the responsive view of your site in any devices, from big tablets to tiny phones.

## ELEMENTS

Sliders, galleries, maps, social, videos, contact forms, buttons, menus, widgets, code, posts, login, tabs & more.

## Spirit Features

> You won't believe what you can accomplish with Spirit

* eCommerce ready
* Built-in MarketPress styles
* Built-in WooCommerce styles
* Clean and crisp design
* Modern typography
* Fully responsive
* Opt-in ready
* Membership ready
* Full-screen images
* Bespoke contact form
* Beautiful galleries
* Custom login form
* Plugin compatible
* Social integrated
* Portfolio focused
* Upfront powered
* Drag and drop
* Front-end text editing
* SEO optimised
* Fast and efficient code


CHANGELOG.md
Change Log
============


1.4.1 - 2017-04-11
-------------------------------------------------------------------------------
- Fix: minor CoursePress compatibility update

1.4 - 2017-03-29
-------------------------------------------------------------------------------
- Add CoursePress support.

1.3 - 2016-11-23
-------------------------------------------------------------------------------
- Add MarketPress support.

1.2 - 2016-10-24
-------------------------------------------------------------------------------
- Add WooCommerce support.

1.1.2 - 2016-04-29
-------------------------------------------------------------------------------
- Fix: visual improvements.

1.1.1 - 2016-04-21
-------------------------------------------------------------------------------
- Fix: styles moving swap-image button to top.

1.1 - 2016-03-23
-------------------------------------------------------------------------------
- Add: full Upfront v1.1 compatibility.

1.0.3 - 2016-02-29
-------------------------------------------------------------------------------
- Visual improvements

- Fix: content alignment on mobile.
- Fix: order of "wellness" on mobile layout.
- Fix: featured image height.
- Fix: image post variations spacing & size.
- Fix: post content lists margin.

1.0.2 - 2016-02-11
-------------------------------------------------------------------------------
- Visual improvements

- Fix: textarea resize and transition.
- Fix: heading size/alignment on responsive contacts page.
- Fix: menu alignment on responsive layouts.
- Fix: hero image height.

1.0.1 - 2016-02-04
-------------------------------------------------------------------------------
- Add: compatibility popup

1.0 - 2016-01-27
-------------------------------------------------------------------------------
- New: compatibility with Upfront 1.0

0.2 - 2015-11-18
-------------------------------------------------------------------------------
- Fix: better alignment on footer elements.
- Fix: breadcrumb style issue.
- Fix: date (visual) on search archive.
- Fix: contact form responsive styles.
- Fix: update responsive navigation.
- Fix: social links vertical alignment.
- Fix: open social media links in new tab.
- Fix: improvements and cleanup on respond form.
- Add: thumbnail to archive (posts list).
- Add: styles to cover unique content.
- Add: image variations and default styles.

0.1.7 - 2015-08-28
-------------------------------------------------------------------------------
- Fix: menu element item deletion positioning.
- Fix: update used paths.
- Fix: button text font style.
- Fix: content editing issue.
- Fix: post image alignment.
- Fix: newsletter form width on responsive.

0.1.6 - 2015-03-05
-------------------------------------------------------------------------------
- Added: default style for tabs element.

0.1.5 - 2015-03-04
-------------------------------------------------------------------------------
- Fix: constact form styles.
- Fix: removed css overriding colors for links and hover states.
- Fix: removed the line-height overriding css.

0.1.4 - 2015-02-23
-------------------------------------------------------------------------------
- Fix: responsive issues in some layouts.
- Fix: button style issues.

0.1.3 - 2015-02-19
-------------------------------------------------------------------------------
- Fix: about page template.
- Fix: post layouts on mobile devices.
- Fix: theme-specific date/time setup.

0.1.2 - 2015-01-29
-------------------------------------------------------------------------------
- Fix: updated styles and typography.

0.1.1 - 2015-01-26
-------------------------------------------------------------------------------
- Second public Beta release.

0.1.0 - 2015-01-21
-------------------------------------------------------------------------------
- Initial public Beta release.


layouts/archive.php
<?php
$layout_version = '1.0.0';

if (file_exists(get_stylesheet_directory() . DIRECTORY_SEPARATOR . 'global-regions' . DIRECTORY_SEPARATOR . 'header.php')) include(get_stylesheet_directory() . DIRECTORY_SEPARATOR . 'global-regions' . DIRECTORY_SEPARATOR . 'header.php');

$region_5 = upfront_create_region(
			array (
  'name' => 'region-5',
  'title' => 'Region 5',
  'type' => 'clip',
  'scope' => 'local',
  'container' => 'region-5',
  'position' => 1,
  'allow_sidebar' => true,
),
			array (
  'row' => 67,
  'breakpoint' => 
  (array)(array(
     'tablet' => 
    (array)(array(
       'edited' => true,
       'col' => 24,
       'background_position_y' => '0',
       'background_style' => 'fixed',
       'background_repeat' => 'no-repeat',
       'background_position' => '50% 0%',
       'background_position_x' => '50',
       'background_type' => 'image',
       'top_bg_padding_num' => '45',
       'top_bg_padding_slider' => '45',
       'row' => 38,
    )),
     'mobile' => 
    (array)(array(
       'edited' => true,
       'col' => 24,
       'background_position_y' => '0',
       'background_style' => 'fixed',
       'background_repeat' => 'no-repeat',
       'background_position' => '50% 0%',
       'background_position_x' => '50',
       'background_type' => 'image',
       'top_bg_padding_num' => '25',
       'top_bg_padding_slider' => '25',
       'row' => 25,
    )),
     'current_property' => 'top_bg_padding_slider',
  )),
  'background_type' => 'image',
  'use_padding' => 0,
  'sub_regions' => 
  array (
    0 => false,
  ),
  'bg_padding_type' => 'varied',
  'top_bg_padding_slider' => '80',
  'top_bg_padding_num' => '80',
  'bottom_bg_padding_slider' => 0,
  'bottom_bg_padding_num' => 0,
  'bg_padding_slider' => 0,
  'bg_padding_num' => 0,
  'background_color' => '#ffffff',
  'background_style' => 'fixed',
  'background_position_y' => '0',
  'background_position_x' => '50',
  'background_image' => '{{upfront:style_url}}/images/archive/bg-archive.jpg',
  'background_image_ratio' => 0.2300000000000000099920072216264088638126850128173828125,
  'background_repeat' => 'no-repeat',
  'background_position' => '50% 0%',
  'version' => '1.0.0',
  'origin_position_y' => '50',
  'origin_position_x' => '50',
  'use_background_size_percent' => '',
  'background_size_percent' => '100',
  'background_default' => 'hide',
  'featured_fallback_background_color' => '#ffffff',
  'background_size' => 'auto',
  'region_role' => 'complementary',
)
			);

$region_5->add_element("PlainTxt", array (
  'columns' => '24',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1453424007894-1113',
  'id' => 'module-1453424007894-1113',
  'options' => 
  array (
    'view_class' => 'PlainTxtView',
    'id_slug' => 'plain_text',
    'content' => '<h1 style="text-align: center;">BLOG</h1>',
    'type' => 'PlainTxtModel',
    'element_id' => 'text-object-1453424007894-1329',
    'class' => 'c24 upfront-plain_txt',
    'has_settings' => 1,
    'padding_slider' => '15',
    'top_padding_num' => '15',
    'bottom_padding_num' => '15',
    'usingNewAppearance' => true,
    'is_edited' => true,
    'use_padding' => 'yes',
    'lock_padding' => '',
    'top_padding_use' => 'yes',
    'top_padding_slider' => '15',
    'bottom_padding_use' => 'yes',
    'bottom_padding_slider' => '15',
    'padding_number' => '15',
    'left_padding_num' => '15',
    'right_padding_num' => '15',
    'anchor' => '',
    'preset' => 'default',
    'row' => 14,
    'current_preset' => 'default',
    'breakpoint' => 
    (array)(array(
       'tablet' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
       'current_property' => 'use_padding',
       'mobile' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
    )),
    'breakpoint_presets' => 
    (array)(array(
    )),
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 0,
  'hide' => 0,
  'toggle_hide' => 1,
  'wrapper_id' => 'wrapper-1453424410671-1492',
  'new_line' => true,
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => false,
      'col' => 12,
      'order' => 1,
      'clear' => true,
    ),
    'mobile' => 
    array (
      'edited' => false,
      'col' => 7,
      'order' => 1,
      'clear' => true,
    ),
    'current_property' => 
    array (
      0 => 'order',
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => false,
      'left' => 0,
      'col' => 12,
      'order' => 0,
    ),
    'mobile' => 
    array (
      'edited' => false,
      'left' => 0,
      'col' => 7,
      'order' => 0,
    ),
    'current_property' => 
    array (
      0 => 'col',
    ),
  ),
));

$regions->add($region_5);

$region_3 = upfront_create_region(
			array (
  'name' => 'region-3',
  'title' => 'Region 3',
  'type' => 'wide',
  'scope' => 'local',
  'container' => 'region-3',
  'position' => 1,
  'allow_sidebar' => true,
),
			array (
  'row' => 10,
  'breakpoint' => 
  (array)(array(
     'tablet' => 
    (array)(array(
       'edited' => false,
       'col' => 24,
    )),
     'mobile' => 
    (array)(array(
       'edited' => false,
       'col' => 24,
    )),
  )),
  'background_type' => 'color',
  'use_padding' => 0,
  'sub_regions' => 
  array (
    0 => false,
  ),
  'bg_padding_type' => 'varied',
  'top_bg_padding_slider' => 0,
  'top_bg_padding_num' => 0,
  'bottom_bg_padding_slider' => 0,
  'bottom_bg_padding_num' => 0,
  'bg_padding_slider' => 0,
  'bg_padding_num' => 0,
  'background_color' => '#ufc6',
  'version' => '1.0.0',
  'region_role' => 'complementary',
)
			);

$region_3->add_element("Uspacer", array (
  'columns' => '1',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1453424059005-1063 upfront-module-spacer',
  'id' => 'module-1453424059005-1063',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'element_id' => 'spacer-object-1453424059004-1273',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 0,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1453424059004-1189',
  'new_line' => true,
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 1,
    ),
    'mobile' => 
    array (
      'col' => 1,
    ),
  ),
));

$region_3->add_element("PlainTxt", array (
  'columns' => '22',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1453423534559-1532',
  'id' => 'module-1453423534559-1532',
  'options' => 
  array (
    'view_class' => 'PlainTxtView',
    'id_slug' => 'plain_text',
    'content' => '<p><a href="{{upfront:home_url}}" target="_self" data-upfront-link-type="homepage">HOME</a> &nbsp;/ &nbsp;BLOG</p>',
    'type' => 'PlainTxtModel',
    'element_id' => 'text-object-1453423534558-1839',
    'class' => 'c24 upfront-plain_txt',
    'has_settings' => 1,
    'padding_slider' => '15',
    'top_padding_num' => '15',
    'bottom_padding_num' => '15',
    'usingNewAppearance' => true,
    'is_edited' => true,
    'use_padding' => 'yes',
    'lock_padding' => '',
    'padding_number' => '15',
    'left_padding_num' => '15',
    'right_padding_num' => '15',
    'anchor' => '',
    'preset' => 'breadcrumbs',
    'breakpoint_presets' => 
    (array)(array(
       'desktop' => 
      (array)(array(
         'preset' => 'breadcrumbs',
      )),
    )),
    'row' => 12,
    'top_padding_use' => 'yes',
    'top_padding_slider' => '15',
    'current_preset' => 'breadcrumbs',
    'breakpoint' => 
    (array)(array(
       'tablet' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
       'current_property' => 'use_padding',
       'mobile' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
    )),
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 0,
  'hide' => 0,
  'toggle_hide' => 1,
  'wrapper_id' => 'wrapper-1453423914639-1759',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => false,
      'col' => 12,
      'order' => 1,
      'clear' => true,
    ),
    'mobile' => 
    array (
      'edited' => false,
      'col' => 7,
      'order' => 1,
      'clear' => true,
    ),
    'current_property' => 
    array (
      0 => 'order',
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'edited' => false,
      'left' => 0,
      'col' => 12,
      'order' => 0,
    ),
    'mobile' => 
    array (
      'edited' => false,
      'left' => 0,
      'col' => 7,
      'order' => 0,
    ),
    'current_property' => 
    array (
      0 => 'col',
    ),
  ),
));

$region_3->add_element("Uspacer", array (
  'columns' => '1',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1453424061271-1304 upfront-module-spacer',
  'id' => 'module-1453424061271-1304',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'element_id' => 'spacer-object-1453424061271-1309',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 0,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1453424061271-1922',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 1,
    ),
    'mobile' => 
    array (
      'col' => 1,
    ),
  ),
));

$regions->add($region_3);

$archive_content = upfront_create_region(
			array (
  'name' => 'archive-content',
  'title' => 'Archive Content',
  'type' => 'wide',
  'scope' => 'local',
  'container' => 'archive-content',
  'position' => 1,
  'allow_sidebar' => true,
),
			array (
  'row' => 80,
  'background_type' => 'color',
  'background_color' => '#ufc6',
  'version' => '1.0.0',
  'breakpoint' => 
  (array)(array(
     'tablet' => 
    (array)(array(
       'edited' => false,
       'col' => 24,
    )),
     'mobile' => 
    (array)(array(
       'edited' => false,
       'col' => 24,
    )),
  )),
  'use_padding' => 0,
  'sub_regions' => 
  array (
    0 => false,
  ),
  'bg_padding_type' => 'varied',
  'top_bg_padding_slider' => 0,
  'top_bg_padding_num' => 0,
  'bottom_bg_padding_slider' => '90',
  'bottom_bg_padding_num' => '90',
  'bg_padding_slider' => 0,
  'bg_padding_num' => 0,
  'region_role' => 'main',
)
			);

$archive_content->add_element("Uspacer", array (
  'columns' => '3',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1453424064057-1988 upfront-module-spacer',
  'id' => 'module-1453424064057-1988',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'element_id' => 'spacer-object-1453424064056-1123',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 0,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1453424064057-1437',
  'new_line' => true,
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 3,
    ),
    'mobile' => 
    array (
      'col' => 3,
    ),
  ),
));

$archive_content->add_element("Posts", array (
  'columns' => '18',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1442668077434-1795',
  'id' => 'module-1442668077434-1795',
  'options' => 
  array (
    'type' => 'PostsModel',
    'view_class' => 'PostsView',
    'has_settings' => 1,
    'class' => 'c24 uposts-object',
    'id_slug' => 'posts',
    'display_type' => 'list',
    'list_type' => 'generic',
    'offset' => 1,
    'taxonomy' => '',
    'term' => '',
    'content' => 'excerpt',
    'limit' => '3',
    'pagination' => 'numeric',
    'sticky' => '',
    'posts_list' => '',
    'thumbnail_size' => 'large',
    'custom_thumbnail_width' => 200,
    'custom_thumbnail_height' => 200,
    'post_parts' => 
    array (
      0 => 'featured_image',
      1 => 'date_posted',
      2 => 'title',
      3 => 'author',
      4 => 'content',
      5 => 'read_more',
    ),
    'enabled_post_parts' => 
    array (
      0 => 'date_posted',
      1 => 'author',
      2 => 'featured_image',
      3 => 'title',
      4 => 'content',
      5 => 'read_more',
    ),
    'default_parts' => 
    array (
      0 => 'date_posted',
      1 => 'author',
      2 => 'gravatar',
      3 => 'comment_count',
      4 => 'featured_image',
      5 => 'title',
      6 => 'content',
      7 => 'read_more',
      8 => 'tags',
      9 => 'categories',
      10 => 'meta',
    ),
    'date_posted_format' => 'd M',
    'categories_limit' => 3,
    'tags_limit' => 3,
    'comment_count_hide' => 0,
    'content_length' => '25',
    'resize_featured' => '1',
    'gravatar_size' => 200,
    'preset' => 'archive',
    'post-part-date_posted' => '<div class="uposts-part date_posted"><span class="date date-day">{{date_1}}</span><span class="date date-month">{{date_2}}</span></div>',
    'post-part-author' => '<div class="uposts-part author">
	Written By <a href="{{url}}">{{name}}</a></div>',
    'post-part-gravatar' => '<div class="uposts-part gravatar">
	{{gravatar}}
</div>',
    'post-part-comment_count' => '<div class="uposts-part comment_count">
	{{comment_count||No comments}}
</div>',
    'post-part-featured_image' => '<div class="uposts-part thumbnail" data-resize="{{resize}}">
	{{thumbnail}}
</div>',
    'post-part-title' => '<div class="uposts-part title">
	<h3><a href="{{permalink}}" title="{{title}}">{{title}}</a></h3>
</div>',
    'post-part-content' => '<div class="uposts-part content {{content_type}}">
	{{content}}
</div>',
    'post-part-read_more' => '<div class="uposts-part read_more">
	<a href="{{permalink}}">Read more</a></div>',
    'post-part-tags' => '<div class="uposts-part post_tags">
	{{tags}}
</div>',
    'post-part-categories' => '<div class="uposts-part post_categories">
	{{categories}}
</div>',
    'post-part-meta' => '<div class="uposts-part meta">
	
</div>
',
    'usingNewAppearance' => true,
    'element_id' => 'module-1442668077434-1795-object',
    'top_padding_use' => 'yes',
    'top_padding_num' => '30',
    'padding_slider' => '15',
    'bottom_padding_num' => '0',
    'use_padding' => 'yes',
    'lock_padding' => '',
    'padding_number' => '15',
    'left_padding_num' => '15',
    'right_padding_num' => '15',
    'anchor' => '',
    'breakpoint_presets' => 
    (array)(array(
       'desktop' => 
      (array)(array(
         'preset' => 'archive',
      )),
       'tablet' => 
      (array)(array(
         'preset' => 'archive-for-tablets',
      )),
       'mobile' => 
      (array)(array(
         'preset' => 'archive-for-mobiles',
      )),
    )),
    'row' => 227,
    'bottom_padding_use' => 'yes',
    'bottom_padding_slider' => '0',
    'top_padding_slider' => '30',
    'current_preset' => 'archive',
    'breakpoint' => 
    (array)(array(
       'tablet' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
       'current_property' => 'use_padding',
       'mobile' => 
      (array)(array(
         'use_padding' => 'yes',
      )),
    )),
    'theme_style' => '',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 0,
  'hide' => 0,
  'toggle_hide' => 1,
  'wrapper_id' => 'wrapper-1442668095508-1224',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 12,
      'order' => 1,
      'clear' => true,
    ),
    'mobile' => 
    array (
      'col' => 7,
      'order' => 1,
      'clear' => true,
    ),
    'current_property' => 
    array (
      0 => 'order',
    ),
  ),
  'breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 12,
      'left' => 0,
      'top' => 0,
      'order' => 0,
    ),
    'mobile' => 
    array (
      'col' => 7,
      'left' => 0,
      'top' => 0,
      'order' => 0,
    ),
    'current_property' => 
    array (
      0 => 'col',
    ),
  ),
));

$archive_content->add_element("Uspacer", array (
  'columns' => '3',
  'margin_left' => '0',
  'margin_right' => '0',
  'margin_top' => '0',
  'margin_bottom' => '0',
  'class' => 'module-1453424066452-1224 upfront-module-spacer',
  'id' => 'module-1453424066452-1224',
  'options' => 
  array (
    'type' => 'UspacerModel',
    'view_class' => 'UspacerView',
    'class' => 'c24',
    'has_settings' => 0,
    'id_slug' => 'uspacer',
    'element_id' => 'spacer-object-1453424066451-1627',
    'preset' => 'default',
    'current_preset' => 'default',
  ),
  'row' => 6,
  'sticky' => false,
  'default_hide' => 1,
  'hide' => 0,
  'toggle_hide' => 0,
  'wrapper_id' => 'wrapper-1453424066451-1154',
  'wrapper_breakpoint' => 
  array (
    'tablet' => 
    array (
      'col' => 3,
    ),
    'mobile' => 
    array (
      'col' => 3,
    ),
  ),
));

$regions->add($archive_content);

if (file_exists(get_stylesheet_directory() . DIRECTORY_SEPARATOR . 'global-regions' . DIRECTORY_SEPARATOR . 'footer.php')) include(get_stylesheet_directory() . DIRECTORY_SEPARATOR . 'global-regions' . DIRECTORY_SEPARATOR . 'footer.php');

